web-client/src/presenter/actions/Public/TrialSessions/getPublicTrialSessionDetailsAction.ts

```diff
@@ -0,0 +1,21 @@
+import { ClientPublicApplicationContext } from '@web-client/applicationContextPublic';
+import { PublicTrialSessionDetails } from '@web-api/business/useCases/trialSessions/getPublicTrialSessionDetailsInteractor';
+
+export const getPublicTrialSessionDetailsAction = async ({
```

Comment:
If you compare with `getTrialSessionDetailsAction.ts`, you'll notice that we are making one less persistence call here. This is because the interactor for `getTrialSessionDetailsAction.ts` does not get swing session details, so the action has to make a separate call for them. In the public case, I just put that swing session fetching logic on the interactor itself.